public/src/components/navbar/NavBarActiveCourse.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react"
+import { Link } from "react-router-dom"
+import { hasTeacher } from "../../Helpers"
+import { useActions, useAppState } from "../../overmind"
+import CourseFavoriteButton from "../CourseFavoriteButton"
+import ToggleSwitch from "./ToggleSwitch"
+
+
+
+const NavBarActiveCourse = (): JSX.Element | null => {
+    const activeCourse = useAppState((state) => state.activeCourse ? (state.courses.find((c) => c.ID === state.activeCourse) ?? null) : null)
```

Comment:
Should we rename `activeCourse` to `activeCourseID`? But we should do it in a separate PR to avoid massive unrelated changes.